runtime/gc_stats.c

```diff
@@ -106,23 +104,31 @@ void caml_orphan_alloc_stats(caml_domain_state *domain) {
 /* The "sampled stats" of a domain are a recent copy of its
    domain-local stats, accessed without synchronization and only
    updated ("sampled") during stop-the-world events -- each minor
-   collection, and on domain termination. */
+   collection, including those happening during domain termination. */
```

Comment:
Is the "including those happening during domain termination" necessary? 

"each minor collection" ==> "minor collection at domain termination"

Reply:
Before this change, the function was called either in STW, or at domain termination (but outside the STW sections of `domain_terminate`). It is important for race-freedom that we only do it during STW sections. It is important for correctness that we do *something* at domain termination, otherwise the sampled copy could remain stale forever after termination, so this is why I point those out explicitly.